sdk/src/ingredient.rs

```diff
@@ -135,30 +152,29 @@ impl Ingredient {
     }
 
     /// Returns an optional Blake3 hash made from the bits of the original image
-    pub fn hash(&self) -> Option<&[u8]> {
-        self.manifest_data.as_deref()
+    pub fn hash(&self) -> Option<&str> {
```

Comment:
I did it for this PR, but for any breaking API change please add the tag `(MINOR)` exactly as typed. This will trigger a minor version bump when we do the next release.

Reply:
Once we go to 1.0, then that rule would change to `(MAJOR)`, but that's not applicable yet.